src/modules/muorb/adsp/px4muorb.cpp

```diff
@@ -48,11 +48,12 @@
 __BEGIN_DECLS
 extern int dspal_main(int argc, char *argv[]);
 extern struct shmem_info *shmem_info_p;
+__END_DECLS
 extern int get_shmem_lock(const char *caller_file_name, int caller_line_number);
 extern void release_shmem_lock(const char *caller_file_name,
 			       int caller_line_number);
 extern void init_shared_memory(void);
```

Comment:
Side note - these externs are kind of horrible. I think we have circular dependencies.
It shouldn't be that hard to tease these pieces apart properly with headers and linking dependencies.

Reply:
Moved those declarations to shmem.h in 178bcd3. It isn't great but it's better than before.